api/nodes/views.py

```diff
@@ -360,7 +364,10 @@ class NodeDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin):
 
     # overrides RetrieveUpdateDestroyAPIView
     def get_object(self):
-        return self.get_node()
+        node = self.get_node()
+        if node.is_registration:
```

Comment:
The NodeList view [also excludes folders and deleted items](https://github.com/CenterForOpenScience/osf.io/pull/4444/files#diff-51eef6ea2a6c35e2b080c9a6fe591e38R220), but the NodeDetail view `get_object` seems to use slightly different rules. Just to check, is that intentional?